boards/arm/96b_argonkey/doc/96b_argonkey.rst

```diff
@@ -0,0 +1,164 @@
+- GPIO with external interrupt capability
+- UART
+- I2C (3)
+- SPI (1)
+- I2S (1)
+
+Supported Features
+==================
+
+The Zephyr 96b_argonkey board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash                               |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+
+More details about the board can be found at `96Boards website`_.
+
+The default board configuration can be found in the defconfig file:
+
+        ``boards/arm/96b_argonkey/96b_argonkey_defconfig``
+
+Connections and IOs
+===================
+
+LED
+---
+
+- LED1 / User1 LED = PB2
+- LED2 / User2 LED = PC13
+
+Push buttons
+------------
+
+- BUTTON = RST (BT1)
+- BUTTON = USR (BT2)
+
+System Clock
+============
+
+96Boards Argonkey can be driven by an internal oscillator as well as the main
+PLL clock. By default System clock is sourced by PLL clock at 84MHz, driven
+by internal oscillator.
+
+Serial Port
+===========
+
+On 96Boards Argonkey Zephyr console output is assigned to USART1.
+Default settings are 115200 8N1.
+
+I2C
+---
+
+96Boards Argonkey board has up to 3 I2Cs. The default I2C mapping for Zephyr is:
+
+- I2C1_SCL : PB6
+- I2C1_SDA : PB7
+- I2C2_SCL : PB10
+- I2C2_SDA : PB9
+- I2C3_SCL : PA8
+- I2C3_SCL : PB4
+
+I2C3 goes to P2 connector and can be used to attach external sensors.
+It goes to 100kbit maximum.
+
+Programming and Debugging
+*************************
+
+Building
+========
+
+Here is an example for building the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: 96b_argonkey
+   :goals: build
+
+Flashing
+========
+
+96Boards Argonkey can be flashed by two methods, one using the ROM
+bootloader and another using the SWD debug port (which requires additional
+hardware).
+
+Using ROM bootloader:
+---------------------
+
```

Comment:
Content missing?

Reply:
For the moment I put it with no content. I will add it next week, if you agree.